src/errors/test-run/templates.js

```diff
@@ -108,6 +120,30 @@ export default {
         ${err.getCallsiteMarkup()}
     `),
 
+    [TYPE.actionElementNonEditableError]: err => markup(err, `
+        The action element is expected to be editable (an input, textarea or element with the contentEditable attribute).
+
+        ${err.getCallsiteMarkup()}
+    `),
+
+    [TYPE.actionElementNonContentEditableError]: err => markup(err, `
+        The action element is expected to have the contentEditable attribute enabled.
```

Comment:
It's also allowed if the document is in `designMode` ([see](https://github.com/DevExpress/testcafe-hammerhead/blob/master/src/client/utils/dom.js#L269)), we check not only `contentEditable` attribute. I think you should discuss it with @VasilyStrelyaev maybe we should update the message